### ISSUE TITLE
Bug in version: 0.1.210 : AttributeError: module 'tensorflow._api.v1.compat.v1.compat' has no attribute 'v1'

### ISSUE THREAD

**Issue by tomaslaz** [google/earthengine-api#127]
It seems that 0.1.210 introduced some changes that break the functionality of the api.
```
> earthengine authenticate
Traceback (most recent call last):
  File "/usr/local/bin/earthengine", line 5, in <module>
    from ee.cli.eecli import main
  File "/usr/local/lib/python3.7/site-packages/ee/cli/eecli.py", line 19, in <module>
    from ee.cli import commands
  File "/usr/local/lib/python3.7/site-packages/ee/cli/commands.py", line 39, in <module>
    tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
  File "/usr/local/lib/python3.7/site-packages/tensorflow_core/python/util/module_wrapper.py", line 193, in __getattr__
    attr = getattr(self._tfmw_wrapped_module, name)
AttributeError: module 'tensorflow._api.v1.compat.v1.compat' has no attribute 'v1'
```

I have tracked them down to:

python/ee/cli/commands.py

L34: import tensorflow.compat.v1 as tf should be import tensorflow as tf

Similarly

python/ee/cli/eecli.py
L91: import tensorflow.compat.v1 as tf should be import tensorflow as tf

**Comment by csaybar** [comment]
Hi @tomaslaz maybe this does not explain the bug, but I had the same error and I solved it by updating my version of TensorFlow.

```
pip3 install tensorflow --upgrade
```

**Comment by bengalin** [comment]
Thanks, we are taking a look.  I believe the problem is in our package and that upgrading tensorflow would not help. @csaybar which versions of tensorflow did you upgrade from and to?

**Comment by csaybar** [comment]
Hi, @bengalin I upgraded TensorFlow from 1.12.0 to 2.1.0.

**Comment by bengalin** [comment]
This should be fixed in version 0.1.212 we just committed.  Please reopen if still an issue after upgrading.
